### DISCUSSION TITLE
SuiteSparse 7.2.1 now in beta

### DISCUSSION THREAD

**Discussion by DrTimothyAldenDavis** [DrTimothyAldenDavis/SuiteSparse#421] [General]
I've enabled the Discussions feature in github for the SuiteSparse repository, which looks like a useful place for comments and discussions like this one.

SuiteSparse 7.2.1 is now in beta release, with various updates, fixes, and comments by @mmuetzel, @kiwifb, @antonio-rojas, @Wimmerer, and @thierry-FreeBSD.  Thanks for all your help.

Let me know when you've had a chance to try out the 7.2.1 release of SuiteSparse.  It should fix the various build issues you've had with 7.2.0 (namely, amd_internal.h not being found by UMFPACK, adding the "None" build type, and others).  Once you've had a chance to try out the 7.2.1.beta1 release, I'll post it as the latest stable release.

This version of SuiteSparse contains GraphBLAS 8.2.1 (also in beta release on the GraphBLAS stand-alone repo).  It implements the v2.0 C API of GraphBLAS.  Once the v2.1 C API is released later in September, I'll be releasing GraphBLAS 9.0.0 along with a corresponding update to SuiteSparse 7.3. GraphBLAS 9.0.0 is also currently in beta release, as a draft implementation of the v2.1 C API, on the GraphBLAS repo (https://github.com/DrTimothyAldenDavis/GraphBLAS/releases/tag/v9.0.0.beta4).

**Comment by antonio-rojas** [comment]
Works fine for me

**Comment by mmuetzel** [comment]
None of the open PRs are blocking a release.
LGTM.

It might be nice to fix #409. But that can wait for 7.3.0 imho.

**Comment by mmuetzel** [comment]
I compiled the beta2 on Windows using the MINGW64 environment of MSYS2 and built Octave with it.
Octave's test suite is still passing.

I also tried to cross-compile from Linux (Ubuntu 23.04) to Windows (mingw-w64 64-bit). That also worked without passing hints that were needed to bypass the compile checks in the previous version.

**Comment by DrTimothyAldenDavis** [comment]
Thanks for all the feedback.  I will post a stable release of v7.2.1 shortly (as soon as the CI finishes).

**Comment by mmuetzel** [comment]
For some reason, `SuiteSparse__thread.cmake` in the 7.2.1 release tarball still contains `check_c_source_runs`. Did something go wrong when merging the branches or did you tag the wrong commit?

**Comment by antonio-rojas** [comment]
The 7.2.1 tag is on the same commit as 7.2.0

**Reply by mmuetzel** [reply]
Yeah. It looks like #437 wasn't merged when the tag was added...

**Comment by DrTimothyAldenDavis** [comment]
Oops -- I just fixed it and reposted v7.2.1 after merging #437 .  Sorry for the confusion.

**Reply by mmuetzel** [reply]
No worries. Thank you for the release.
Looks good now. 👍
